app/src/main/java/space/taran/arknavigator/ui/fragments/ResourcesFragment.kt

```diff
@@ -57,7 +66,7 @@ class ResourcesFragment : MvpAppCompatFragment(), ResourcesView {
 
     private lateinit var binding: FragmentResourcesBinding
     private var resourcesAdapter: ResourcesRVAdapter? = null
-
+    var selectedTag: Tag = ""
```

Comment:
If you want to initialize Resources screen with tag from Gallery, I would use nullable type and `null` instead of `""`. But even better would be to allow passing query instead of single tag (can be useful in future, when we implement saved queries). So type should be something like `Tags?`. `Tags` itself is alias to `Set<Tag>` defined in `Tags.kt`.